internal/rbd/rbd_util.go

```diff
@@ -127,6 +127,12 @@ type rbdImage struct {
 	RequestName string
 	NamePrefix  string
 
+	// ParentName represents the parent image name of the image.
+	ParentName string
+	// Parent Pool is the pool that contains the parent image.
+	ParentPool      string
+	ImageFeatureSet librbd.FeatureSet
```

Comment:
why export `ImageFeatureSet`?

Reply:
without this one, the golint was failing and most of the members of rbdImage are exported and it's for a reason?

Reply:
I hope that in the future we do not need to have the members exported, as that invites to not have a nice `rbdImage` API. It is fine for now.